python/hail/dataset.py

```diff
@@ -234,7 +234,7 @@ def aggregate_intervals(self, input, expr, output):
 
         - ``interval`` (*Interval*): genomic interval
         - ``global``: global annotations
-        - ``variants`` (*Aggregable[Variant]*): aggregable of :ref:`variant`s Aggregator namespace below.
+        - ``variants`` (*Aggregable[Variant]*): aggregable of :py:class:`~hail.representation.Variant` s Aggregator namespace below.
```

Comment:
Not for you to fix with this PR -- we should come to a consensus on whether we want to refer to the Python type or the expression language type (coming soon).